tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -690,13 +654,15 @@ describe(__filename, () => {
         undefined,
         loadAddonsByAuthors({
           addons: Object.values(addons),
-          authorUsernames: ['test', 'test2'],
+          authorIds: [fakeAuthorOne.id, fakeAuthorTwo.id],
           count: Object.values(addons).length,
           pageSize: EXTENSIONS_BY_AUTHORS_PAGE_SIZE,
         }),
       );
 
-      expect(getAddonsForUsernames(state, ['test', 'test2'])).toEqual([
+      expect(
+        getAddonsForAuthorIds(state, [fakeAuthorOne.id, fakeAuthorTwo.id]),
```

Comment:
I find this similarly confusing, although not as bad as the one I commented on above. I guess we're supposed to assume that `firstAddon` has `fakeAuthorOne` as an author and `secondAddon` has `fakeAuthorTwo` as an author?

I realize you probably didn't write these tests and are just refactoring some code, but I am disturbed by how opaque the set up is for many of these tests.

Reply:
> I realize you probably didn't write these tests and are just refactoring some code, but I am disturbed by how opaque the set up is for many of these tests.

I did not write any of these test cases indeed. It was already painful to adapt them... 😢

Reply:
I am going to correct everything you mentioned in this review, but I will leave other improvements for the next PR or next code quality PR, because this PR is quite big already.

Reply:
I found a way to be more verbose in those test cases, I haven't replaced everything, but I could do that in a follow-up or in the next PR (https://github.com/mozilla/addons-frontend/pull/6686) maybe.